README.md

```diff
@@ -119,6 +119,20 @@ We welcome community contributions to this project.
 
 Please read our [Contributor Guide](CONTRIBUTING.md) for more information on how to get started.
 
```

Comment:
Can you move this section up just before the "Contributing" section?

Reply:
Moved the section before the contributing section